### DISCUSSION TITLE
False Positives on BT store for PS5

### DISCUSSION THREAD

**Discussion by thetallchap** [jef/streetmerchant#2830] [Help]
Hi, has anyone overcome the false positives for PS5 on the BT store please?
Thx

**Comment by Mr-Tech-13** [comment]
I'll take a look when I get home, but I think bestbuy changed some code
EDIT: LOL, I saw BB not BT, I am blind.

**Comment by Mr-Tech-13** [comment]
It doesn't look like anything has changed

**Comment by thetallchap** [comment]
On the British Telecom - BT store? Ok many thanks for checking and apologies for wasting your time.

Regards

Jon Harris

Sent from my iPhone

On 3 Nov 2021, at 21:10, Mr-Tech-13 ***@***.***> wrote:

﻿

It doesn't look like anything has changed

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/jef/streetmerchant/discussions/2830#discussioncomment-1584900>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATF43OMRE2CMIGGGYJFRL2TUKGXOBANCNFSM5G6W4NLA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Mr-Tech-13** [reply]
No problem!
